app/uk/gov/hmrc/customs/declaration/controllers/FileUploadController.scala

```diff
@@ -57,9 +61,11 @@ extends BaseController {
       logger.debug(s"Request received. Payload = ${vupr.body.toString} headers = ${vupr.headers.headers}")
 
         fileUploadBusinessService.send map {
-          case Right(_) =>
+          case Right(res) =>
             logger.info(s"Upload initiate request processed successfully")
-            Accepted.as(MimeTypes.XML).withConversationId
+            val request: InitiateUpscanUploadRequest = res.uploadRequest
```

Comment:
what happen when XML is not valid?

Reply:
We have a much more serious problem, we construct the XML ourselves so it should always be valid.